.github/workflows/ensure_pull_requests_are_labeled.yml

```diff
@@ -0,0 +1,28 @@
+
+name: Ensure pull requests are labeled
```

Comment:
It's intentional that _you_, the PR opener, can't label your own PR. That's fine! This is a reminder to the _maintainers_ that they must label the PR before merging.

So, with that in mind, maybe the name should be something like:

```Maintainer must label all pull requests```